### PR TITLE
changed visibility of Featured Subsite paragraph type labels from visally-hidden to hidden

### LINKED ISSUE #132: Labels for Featured Subsite paragraph type should be hidden, not visually hidden
Labels for fields in the Featured Subsite paragraph type are currently visually hidden:

Screenshot 2021-06-24 at 08 55 58

To prevent them from being announced to screen readers or appearing in an unstyled context they should be hidden.

### PR DESCRIPTION
**Pull request by bisd**
Please review this pull request. Thank you.

### CODE REVIEW

**Review by markconroy — COMMENTED** [review]
Hi @bisd 

Thanks for this. This PR needs to be made against the localgov_paragraphs repo, not subsites.

File: config/optional/core.entity_view_display.paragraph.localgov_featured_subsite.default.yml [hunk 1]
```diff
@@ -0,0 +1,55 @@
+uuid: 5e3182be-026a-4bb1-9523-afcff0640504
```
Comment (markconroy) [inline config/optional/core.entity_view_display.paragraph.localgov_featured_subsite.default.yml:1]: We need to remove this UUID line from the code.

File: config/optional/core.entity_view_display.paragraph.localgov_featured_subsite.default.yml [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+  config:
+    - field.field.paragraph.localgov_featured_subsite.localgov_featured_subsite_image
+    - field.field.paragraph.localgov_featured_subsite.localgov_featured_subsite_link
+    - field.field.paragraph.localgov_featured_subsite.localgov_featured_subsite_text
+    - paragraphs.paragraphs_type.localgov_featured_subsite
+  module:
+    - field_formatter_class
+    - link
+_core:
+  default_config_hash: 8_XNHbVXftK6vOq0-Ynbao8kphjlrCcAT8tXqsYIggk
```
Comment (markconroy) [inline config/optional/core.entity_view_display.paragraph.localgov_featured_subsite.default.yml:14]: These 2 lines can be removed

**Review by markconroy — CHANGES_REQUESTED** [review]

File: config/optional/core.entity_view_display.paragraph.localgov_featured_subsite.default.yml (hunk 1 above)
Comment (bisd) [reply config/optional/core.entity_view_display.paragraph.localgov_featured_subsite.default.yml:1]: Thanks @markconroy the localgov_paragraphs repo does not contain a config/options folder, only a config/install. Should I create a config/options folder in the localgov_paragraphs repo to put this file in?

Comment (msayoung) [reply config/optional/core.entity_view_display.paragraph.localgov_featured_subsite.default.yml:1]: @bisd - this is the file you are editing https://github.com/localgovdrupal/localgov_paragraphs/blob/2.x/modules/localgov_subsites_paragraphs/config/install/core.entity_view_display.paragraph.localgov_featured_subsite.default.yml 

It's in a submodule of the localgov_paragraphs module. there is no config/optional directory here, config/install is fine.

**Comment by bisd** [comment]
I am closing this pull request as the pull request needs to be made against the localgov_paragraphs repo, not localgov_subsites. I have made a new pull request at https://github.com/localgovdrupal/localgov_paragraphs/pull/133

**Pull request closed without merging**